projects/components/src/textarea/textarea.component.ts

```diff
@@ -29,6 +30,9 @@ export class TextareaComponent implements OnInit {
   @Input()
   public disabled: boolean | undefined;
 
+  @Input()
+  public rows: number = -1;
```

Comment:
is '-1' a legit input for `mat text area`? It seems odd.

Reply:
what if we keep the default value of rows to '0' ?

Reply:
I didn't have problems with this value. I did with 0. Probably I should use a positive value as a default.

Reply:
If I use '0' the test threw an " IndexSizeError " DOMException

Reply:
or just `undefined` should let the browser default hold, right?

Reply:
Good advise!. Done

Reply:
Using rows as undefined generates  the next error

Reply:
I think the best option is to use 2 as default since it is the default that textarea has by definition.